frontend/src/components/maps/BuildingPopupView.tsx

```diff
@@ -31,31 +37,39 @@ export const BuildingPopupView: React.FC<IBuildingDetailProps> = ({ building })
               <ListGroup.Item>
                 <div>
                   <Label>Address: </Label>
-                  {building?.address?.line1}
+                  {buildingDetail?.address?.line1}
                 </div>
                 <div>
-                  {building?.address?.city}, {building?.address?.province}{' '}
-                  {building?.address?.postal}
+                  {buildingDetail?.address?.city}, {buildingDetail?.address?.province}{' '}
+                  {buildingDetail?.address?.postal}
                 </div>
               </ListGroup.Item>
             </ListGroup>
             <ListGroup>
               <ListGroup.Item>
                 <Label>Floor Count: </Label>
-                {building?.buildingFloorCount}
+                {buildingDetail?.buildingFloorCount}
               </ListGroup.Item>
               <ListGroup.Item>
                 <Label>Predominate Use: </Label>
-                {building?.buildingPredominateUse}
+                {buildingDetail?.buildingPredominateUse}
               </ListGroup.Item>
-              {building?.projectNumber && (
+              {buildingDetail?.projectNumber && (
                 <ListGroup.Item>
                   <Label>RAEG or SPP: </Label>
-                  {building?.projectNumber}
+                  {buildingDetail?.projectNumber}
                 </ListGroup.Item>
               )}
             </ListGroup>
           </Col>
+          <Col>
+            {!props?.disabled &&
+              (!keycloak.hasAgency(buildingDetail?.agencyId) ? (
+                <Link to={`/submitProperty/${buildingDetail?.parcelId}?disabled=true`}>View</Link>
```

Comment:
I'm curious about the syntax here.  Are we changing the url to include `?disabled=true`?  If so, that probably isn't the most ideal way to control the view vs edit mode.

Also please change the route from `/submitProperty` to `/property`.

Reply:
Not sure which approach we want to take, was just keeping this popup consistent with the ParcelPopupView. But will do re the route!

Reply:
I've seen this and the other popup where "disabled" means "view" - I'd like to suggest that we use "mode" instead. What does the team think?

`mode = "view"` -> view-mode form
`mode = "edit"` -> edit-mode

this would support future modes (if they become a requirement)

`mode = "admin-mode-that-can-impersonate-users"` -> admin-mode form

Reply:
I like it

Reply:
I'm okay with the URL having `?mode=edit` because I would expect the component (or route or somewhere) would check that the user has the appropriate permission to display the form. If user cannot edit, then I would show the view mode. If user cannot even view properties (we have a permission/claim for that as well), then it would go to the "permission-denied" page

Reply:
I would prefer `mode` as well.